### PR TITLE
Update email

### PR DESCRIPTION
**Pull request by lukasbuenger**
Adds a new option to change your email address to the Account section. You immediately will get logged out afterwards.

### CODE REVIEW

**Comment by lukasbuenger** [comment]
image

Isolated

File: components/Account/UpdateMe.js [hunk 1]
```diff
@@ -150,10 +151,10 @@ class UpdateMe extends Component {
     const errorMessages = Object.keys(errors)
       .map(key => errors[key])
       .filter(Boolean)
-
     return (
       <Loader loading={loading || !me} error={error} render={() => (
         <div style={style}>
+          <UpdateEmail t={t} me={me} />
```
Comment (tpreusse) [inline components/Account/UpdateMe.js]: nit: you already have `t` from `withT`, use `withMe` in `UpdateEmail` and make it completely prop less.

Comment (lukasbuenger) [reply components/Account/UpdateMe.js]: Will that not trigger an unnecessary query?

Comment (tpreusse) [reply components/Account/UpdateMe.js]: apollo does query de-deup—will just load it from the in-memory cache

Comment (lukasbuenger) [reply components/Account/UpdateMe.js]: And I'm not sure whether "sign out via refetch" still would work.

**Pull request merged**